### PR TITLE
Added new texts for UR and LT

### PR DESCRIPTION
**Pull request by NC-001**
- Added missing texts for UR and LT

### CODE REVIEW

File: NDB.Covid19/NDB.Covid19/Locales/lt.json [hunk 1]
```diff
@@ -259,7 +259,7 @@
   "INFOCONSENT_LOGIN": "Sutinku ",
   "REGISTER_ERROR_NOMATCH_HEADER": "Jūsų testo rezultatas nerastas",
   "REGISTER_CONTINUE_WITH_SELF_TEST_BUTTON_TEXT": "Patvirtinu, kad mano testo rezultatas buvo teigiamas ir pranešiu, kad esu užsikrėtęs",
-  "REGISTER_ERROR_NOMATCH_DESCRIPTION": "<p>Gali praeiti šiek tiek laiko, kol rezultatas bus atnaujintas MSIS. Bandykite dar kartą vėliau. Norėdami pasidalinti savo testo rezultatais su „Smittestopp“, turite būti vyresni nei 16 metų.",
+  "REGISTER_ERROR_NOMATCH_DESCRIPTION": "<p> Rezultatų atnaujinimas MSIS gali užtrukti. Galite bandyti vėliau. </p> Jei galite patvirtinti, kad testavimo stotyje arba savikontrolės metu gavote teigiamą testo rezultatą, vis tiek galite pranešti apie infekciją paspausdami apačioje esantį viršutinį mygtuką.",
```
Comment (pavshr) [inline NDB.Covid19/NDB.Covid19/Locales/lt.json]: Is there an extra space?

Comment (NC-001) [reply NDB.Covid19/NDB.Covid19/Locales/lt.json]: extra space where?

Comment (pavshr) [reply NDB.Covid19/NDB.Covid19/Locales/lt.json]: `<p> Rez`

Comment (NC-001) [reply NDB.Covid19/NDB.Covid19/Locales/lt.json]: fixed it

**Review by pavshr — APPROVED** [review]

**Pull request merged**